src/Illuminate/Http/Request.php

```diff
@@ -255,6 +255,24 @@ public function has($key)
 	}
 
 	/**
+	 * Determine if the request contains a non-empty value for an input item matching a given pattern.
+	 *
+	 * @param  string  $pattern
+	 * @return bool
+	 */
+	public function hasRegex($pattern)
+	{
+		$input = $this->all();
+
+		foreach ($input as $key => $value)
```

Comment:
@phpeek  Well, I can change, the example I showed you has a different context.

What you think about that @GrahamCampbell?